superset/initialization/__init__.py

```diff
@@ -48,6 +48,8 @@
     results_backend_manager,
     talisman,
 )
+from superset.models.superset_core.role import SupersetRole
```

Comment:
you should not import models here, it could leverage a problems when imports models too soon